docs/index.md

```diff
@@ -1,17 +1,101 @@
 # Welcome to whisk's documentation!
 
-TODO: introduction to whisk
+**[whisk](https://github.com/whisk-ml/whisk) is a ML project framework that make makes collaboration, reproducibility, and deployment "just work".**
+
+Tying together the tools required to release a machine learning model can be daunting. Whisk makes building and releasing ML models easy and fun. Whisk creates a logical and flexible project structure for ML with reproducible results and lets you release your model to the world without becoming a software engineer.
+
+Whisk doesn't lock you into a particular ML framework or require you to learn yet another ML packaging API. Instead, it leverages the magic of Python's ecosystem that's available to projects structured in a Pythonic-way. Whisk does the structuring while you focus on the data science.
+
+Read more about our [beliefs](#beliefs).
 
 ## Quickstart
 
-TODO
+_Replace `demo` with the name of your ML project in the examples below._
+
+### Create a project for the model
+
+Create the project with whisk:
+
+```
+$ pip install whisk
+$ echo "Generate the directory structure, set up a venv, initialize a Git repo, and more."
+$ whisk create demo
+$ cd demo
+$ source venv/bin/activate
+```
+
+Take a quick tour the project you just created:
+
+```
+$ jupyter-notebook notebooks/getting_started.ipynb
+```
+
+The notebook shows how to save your trained model to disk, use the saved model to generate predictions, how to load Python functions and classes from the project's `src` directory for a cleaner notebook, and more. It's the guide rails for your own ML project.
+
+### Test the model
```

Comment:
I think we need to be careful about using the word "test". For a developer, "test" means actual testing code. I think think a more accurate word for this section title might be "Try".

A developer might see this section and say "they think that is a test?".

We do actually have pre-baked tests baked in to the project as well.

Reply:
Ah yeah great call-out. I like "Try" much better - Ill see how that looks!

Reply:
I addressed this in my PR